src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/ModuleWellKnownAttributeData.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Information decoded from well-known custom attributes applied on a module.
+    /// </summary>
+    internal sealed class ModuleWellKnownAttributeData : CommonModuleWellKnownAttributeData
+    {
+        #region SkipLocalsInitAttribute
```

Comment:
Might as well drop the region. There's nothing else in this file.

Reply:
I believe the point in keeping the region is to make sure future changes in this file keep the pattern. For instance, before I included SkipLocalsInitAttribute in `TypeWellKnownAttributeData`, there was only one region there (CoClassAttribute).